tests/unit/fem/test_pa_coeff.cpp

```diff
@@ -19,6 +19,29 @@ namespace pa_coeff
 
 int dimension;
 
+Mesh MakeCartesianNonaligned(const int dim, const int ne)
```

Comment:
We can also do random perturbations to the vertices -- that should give us even more general shapes. No need to do it now though, maybe some other time.

Reply:
I also ran the tests using some other unstructured/severely warped meshes, and everything passed 👍

Reply:
Excellent, thank you @pazner !